code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -58,7 +58,8 @@
 /mob/living/carbon/human/monkey
 	gender = PLURAL
 
-/mob/living/carbon/human/monkey/Initialize()
+/mob/living/carbon/human/monkey/Initialize(mapload, species_name, datum/dna/new_dna, decl/bodytype/new_bodytype)
 	if(gender == PLURAL)
 		gender = pick(MALE, FEMALE)
-	. = ..(species_name = SPECIES_MONKEY)
```

Comment:
FWIW, this specific pattern was the only thing that dropped all the other arguments, I think. Most of the other overrides don't technically need the arguments but there's no reason not to have them now that you've added them.